test/fixtures/node_modules/no_exports/index.js

```diff
@@ -0,0 +1 @@
+export default 'main'
```

Comment:
This should probably be:
```suggestion
export default 'index'
```

Since the `main` field is undefined.